_proto/hapi/chart/metadata.proto

```diff
@@ -17,6 +17,10 @@ message Maintainer {
 //
 // 	Spec: https://k8s.io/helm/blob/master/docs/design/chart_format.md#the-chart-file
 message Metadata {
+	enum Engine {
+		// gotpl is the default engine
+		gotpl = 0;
```

Comment:
I added the Engine type just a few weeks ago, and I think it's only used in one place in the code. I would be happy to change that to use the Enum you supplied here. We reserved the right to  change the protos (in a backward compatibility breaking way) up until Beta 1, so we could do that if it's a better fit.